scripts/src/EncryptedStorageProviders/utils.ts

```diff
@@ -0,0 +1,2 @@
+const KIN_WALLET_STORAGE_INDEX = 'kin-wallet';
+export const getIndex = (id?: string) => `${KIN_WALLET_STORAGE_INDEX}${id && `_${id}`}`;
```

Comment:
Just verifying my thoughts: this opens the option for *multiple accounts*?

Reply:
@katzio it does if an ID is passed